test/yjsEditorTest.ts

```diff
@@ -0,0 +1,156 @@
+import { Editor, Operation } from 'slate';
+import * as Y from 'yjs';
+import { applySlateOps as applySlateOperations } from '../src/apply';
+import { toSlateOps } from '../src/convert';
+import { SyncDoc, SyncElement } from '../src/model';
+
+export interface YjsEditorTest extends Editor {
+  isRemote: boolean;
+  doc: Y.Doc;
+  syncDoc: SyncDoc;
+  shouldCaptureYjsUpdates: boolean;
+  capturedYjsUpdates: Uint8Array[];
+  onChangeComplete: () => void;
+}
+
+export const YjsEditorTest = {
```

Comment:
Just too keep things consistent, I would suggest renaming this to YjsTestEditor

Reply:
Done.